docs/index.md

```diff
@@ -33,3 +33,13 @@ Date         | Message
 Nov 20, 2017 | New RINGMesh website !
 </div>
 
+<div class="col-md-12" news-table markdown="1">
+## They are already using it!
+<div class="logo_users">
+<div></div>
```

Comment:
Why do you need his empty div?

Reply:
to have fake spaces before and after the logo bar